llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -748,7 +749,7 @@ Error InstrProfSymtab::create(const FuncNameIterRange &FuncIterRange,
   return Error::success();
 }
 
-void InstrProfSymtab::finalizeSymtab() {
+void InstrProfSymtab::finalizeSymtab() const {
```

Comment:
This seems like an odd one to be marked const, as the name implies mutation.

Reply:
added a comment

Reply:
The description of the method says "If the symtab is created by a series of calls to \c addFuncName, \c
  // finalizeSymtab needs to be called before looking up function names". This to me makes it sound like it is a user-visible action, notifying the symtab that the user is done adding functions. The usage is a little confusing though - it is called by both the create methods (clearly mutable), and by some get* methods (now non-mutable). Maybe the comment needs to be clarified?